apis/wikis.es6.js

```diff
@@ -0,0 +1,31 @@
+import { runQuery } from '../apiBase/APIRequestUtils';
+import Wiki from '../models2/Wiki';
+
+const getPath = (query) => {
```

Comment:
ssh: Could not resolve hostname dont tell my wife Im: nodename nor servname provided, or not known